common/include/Utilities/FolderUtils.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+#include <fstream>
+#include <iostream>
+#include <iomanip>
+#include "FixedPointTypes.inl"
+#include "ghc/filesystem.h"
+#include "Path.h"
+
+namespace fs = ghc::filesystem;
+
+
+namespace Path
+{
+    extern bool IsRelative(const std::string& path);
+
+    /************************************************************************
+     * @brief Indicates whether dir2 is contained within dir1
+     * @param dir1 Directory that may or may not contain the second directory
+     * @param dir2 The second directory
+     * @return true/false
+     *************************************************************************/
```

Comment:
I'd try and keep comments styles consistent within the codebase.
I personally haven't seen this type before.

Reply:
It looks like it's just doxygen with the extra row of astrix.  imo I like them since VS now supports them in the latest release, but if we are going to use doxygen, they should be used across the entire namespace.